service/header/p2p_exchange.go

```diff
@@ -98,11 +99,10 @@ func (ex *P2PExchange) performRequest(ctx context.Context, req *pb.ExtendedHeade
 	if req.Amount == 0 {
 		return make([]*ExtendedHeader, 0), nil
 	}
-	if ex.trustedPeer == "" {
-		return nil, fmt.Errorf("no trusted peer")
-	}
```

Comment:
`if len(ex.trustedPeer) == 0` ?

Reply:
I thought now we are using peers by default if nothing were provided...

Reply:
Yes we are, but it is still possible to import P2PExchange from another project and give zero peers.
Think of packages inside ‘service’(will probably get extracted soon) as libraries that eventually will be used by other people and not just by us.

Reply:
condition added